etc/di.xml

```diff
@@ -19,4 +19,9 @@
             <argument name="catalogSearchEvent" xsi:type="object">Metrilo\Analytics\Model\Events\CatalogSearch\Proxy</argument>
         </arguments>
     </type>
+    <type name="Metrilo\Analytics\Block\Analytics">
```

Comment:
Why do you need this pattern for this class? It doesn't load anything in the constructor?

Reply:
No idea what i was thinking tbh... Will remove it.